### PR TITLE
Update unit test data, Add totalAmount to cert

### PR DESCRIPTION
**Pull request by yurikoinaba**
- Update unit test data to use transactions from testnet (with some exceptions - MC Backward Transfer Request, Ceased Sidechain Withdrawal still uses regtest data)
- Rename `isFromBackwardTransfer` to `backwardTransfer` and remove `pubKeyHash` from certificate output (to match block explorer response. ex https://explorer-testnet.horizen.io/api/tx/410a983c24f14bc0c2bef11c3274f399b99f32235742da8b3eca1ebb2986146f)
- Some edits to `test/helper/formatCertificate.js`
- Add `totalAmount` (`The total value of the certificate in ZEN`) to `cert`

### CODE REVIEW

File: src/certificate.js [hunk 1]
```diff
@@ -64,6 +64,17 @@ function deserializeCertFields (buf: Buffer, offset: number) {
     return { cert, offset }
   }
 
+  function getTotalAmount(outputs) {
```
Comment (loucaponi) [inline src/certificate.js:67]: Perhaps a more specific function name?  The logic is specific to backward transfers, so maybe `getTotalBtAmount`?

**Review by loucaponi — COMMENTED** [review]
Looks great, left a few comments to consider, but no show stoppers.  Let me know what you decide, and I'll approve when ready

**Review by loucaponi — APPROVED** [review]

**Comment by yurikoinaba** [comment]
@cronicc It looks like we need to run the test with `export NODE_OPTIONS=--openssl-legacy-provider` for Node v17
https://github.com/webpack/webpack/issues/14532

**Comment by cronicc** [comment]
> @cronicc It looks like we need to run the test with `export NODE_OPTIONS=--openssl-legacy-provider` for Node v17 [webpack/webpack#14532](https://github.com/webpack/webpack/issues/14532)

Addressed in https://github.com/HorizenOfficial/zencashjs/pull/67, in the long run it might be better to switch to a version of webpack that doesn't have that issue (if there is one).

**Review by paolocappelletti — APPROVED** [review]

**Review by cronicc — APPROVED** [review]

**Pull request merged**